src/qibolab/instruments/qm/ports.py

```diff
@@ -149,7 +180,7 @@ class OctaveOutput(QMOutput):
 
     Can be external or internal.
     """
-    output_mode: str = field(default="triggered", metadata={"config": "output_mode"})
+    output_mode: str = field(default="always_on", metadata={"config": "output_mode"})
```

Comment:
Would you remind me why you changed default?

Reply:
The triggered mode requires some calibration to work correctly, see: https://github.com/qiboteam/qibolab/blob/6983b8834546f95d2b78af9ee37d192c31aee88e/src/qibolab/instruments/qm/ports.py#L4-L11)

These numbers seem to be valid for OPX+, but I tested and they don't work properly in the OPX1000. There are other options to keep using triggered: try to find the OPX1000 parameters somewhere in the docs (if they are there) or calibrate them ourselves, however just switching to `always_on` was the quickest solution.

The side-effect of this is that Octave LOs will remain on forever, even after program execution. At least this is what we observed with the OPX+ cluster and earlier QUA versions, that's why I preferred using `triggered`, but I have not checked if this is still the case for the OPX1000. This may cause slight temperature increase if high gains are used. My temporary solution to protect against this is to have a cronjob every 20 minutes that runs a minimal program that turns everything to `always_off`.

Reply:
> This may cause slight temperature increase if high gains are used. My temporary solution to protect against this is to have a cronjob every 20 minutes that runs a minimal program that turns everything to `always_off`.

Ok, maybe we should start storing this type of hints somewhere. Either keep it in an issue, or you could use https://github.com/qiboteam/qibolab/wiki (or even a markdown note stored directly in the repo, maybe below `doc/notes`, or something like that).

Reply:
Regarding storing hints, another thing to point out is that using the OPX1000 (at least the one in TII), requires the qm-qua 1.2.1a2 pre-release. However, I have not updated the pyproject because this would most likely break the OPX+ cluster which requires 1.1.6. As discussed earlier, I am not sure if there is a way to have different versions of the same library in different optional dependency groups in poetry. If no, we should find a better way to communicate this.

Regarding the LO mode discussed above, I believe an issue would be sufficient.

Reply:
So, this is the official specification of the dependency format in the distributed package
https://packaging.python.org/en/latest/specifications/dependency-specifiers/
(in our case the package is built by Poetry, but it's less relevant who is building than who will consume it, i.e. `pip` - even though also Poetry itself when managing the env, but in a compatible way).

Despite not being written explicitly, I believe it is assumed that a name appears only once (we could try what happens if we specify it twice, but it will either complain, or overwrite).
In any case, this is not an issue we could face, since we're not writing this file manually, but it is written by the Poetry build backend from the `pyproject.toml` file, which could be considered essentially a dictionary (and most likely it's somewhere deserialized to it), and repeating the same name multiple times would lead to a clash in the keys, so we know it won't be supported.

What we could do is to circumvent the limitation, and instead of listing just the name and version of the package (assuming it will be fetched from PyPI), we could try to assign it an arbitrary name, and use directly the URL to specify which package to use (so, we could name the package `qm-qua-1.2.1a2`, since the distribution name and import name do not have to match).
In case there is some requirement that the package name should be matched with the metadata, we could download the QUA package from PyPI, and reupload it on GitHub, patching the metadata, pointing the URL there.

So, I believe we could work around the limitation, and depend on both versions "somehow". But it'd be a patch. Whether it's worth or not has to be established (in a sene: you tell me).

Reply:
> Whether it's worth or not has to be established (in a sense: you tell me).

I don't think it's worth the effort, given that it is expected to be temporary. Hopefully, qm-qua will end up with an official release that will work with both cluster soon (I just have no idea how soon). Until then, it will probably take less effort to open an issue and communicating this to the people interested. So far it is a localized TII issue, so they are not that many, and most of them are already aware.

The only place where some work around may be needed at some point is the CI, but I would postpone that until we actually have proper QM tests. Most likely we won't even have the multiple version issue there, since these tests will be instrument (and firmware)-free so they should not have version limitations.

In summary, I will advance a bit with the 0.2 version (#1068) to check something and if everything works well there as well, I will open the two issues discussed here and merge this PR (also after a second approval).